distsql/distsql.go

```diff
@@ -205,9 +205,10 @@ func (r *selectResult) readRowsData(chk *chunk.Chunk) (err error) {
 	rowsData := r.selectResp.Chunks[r.respChkIdx].RowsData
 	maxChunkSize := r.ctx.GetSessionVars().MaxChunkSize
 	timeZone := r.ctx.GetSessionVars().GetTimeZone()
+	var bufferedBytes []byte
```

Comment:
s/bufferedBytes/buf/

Reply:
Why not save the buf in the selectResult?